spec.html

```diff
@@ -8274,7 +8274,14 @@
         <h1>[[DefineOwnProperty]] ( _P_, _Desc_ )</h1>
         <p>When the [[DefineOwnProperty]] internal method of a module namespace exotic object _O_ is called with property key _P_ and Property Descriptor _Desc_, the following steps are taken:</p>
         <emu-alg>
-          1. Return *false*.
+          1. Let _current_ be ? _O_.[[GetOwnProperty]](_P_).
+          1. If _current_ is *undefined*, return *false*.
+          1. If IsAccessorDescriptor(_Desc_) is *true*, return *false*.
+          1. If _Desc_.[[Writable]] is present and has value *false*, return *false*.
```

Comment:
It wasn't clear to me how to simplify things but in the meantime I think I found a way to factor out the common logic (something like IsSameDescriptor doesn't cut it, I believe). Will update the PR soon.